frontend/web/components/release-pipelines/CreateReleasePipeline.tsx

```diff
@@ -23,9 +27,24 @@ const blankStage: DraftStageType = {
   },
 }
 
+type CreateReleasePipelineParams = {
+  projectId: string
+  id?: string
+}
+
 function CreateReleasePipeline() {
   const history = useHistory()
-  const { projectId } = useParams<{ projectId: string }>()
+  const { id: pipelineId, projectId } = useParams<CreateReleasePipelineParams>()
+
+  const { data: existingPipeline } = useGetReleasePipelineQuery(
```

Comment:
NIT but for UX we have a little lag before the data arrives so we could add a loader